src/hub.html

```diff
@@ -226,6 +226,17 @@
                     listed-media
                 >
                     <a-entity class="freeze-menu" visibility-while-frozen="withinDistance: 100;">
+                        <a-entity mixin="rounded-text-button ui"
```

Comment:
Weirdest part of this is that we want to have a networked refresh for admins and a local refresh for non-admins when there's a media-image that results from a twitch.tv url (e.g. the "stream offline" case). We don't give non-admins the networked refresh button because they aren't permitted to send NAF traffic for the object. This is fine, but means that admins have both buttons - networked refresh on hover and local refresh in the freeze menu.